src/main/resources/ListedPrintMonograph.json

```diff
@@ -208,7 +208,7 @@
         {
           "field": "RECEIVING_NOTE",
           "dataSource": {
-            "from": "//LocalData[Description='LocalData2']/Value"
+            "from": "//LocalData[Description='LocalData4']/Value"
```

Comment:
this statement should be added, see jira ticket for details
"dataSourceFieldName" : "Local Data 4"